www/app/api/chat/honcho/route.ts

```diff
@@ -17,14 +27,19 @@ export async function POST(req: NextRequest) {
 
   const { appId, userId } = userData;
 
+
+
+  const query = `Given the following user message: <user>${message}</user> I had the following message: ${parseHonchoContent(thought)}`
```

Comment:
can we change the second "message" to query? the user sends a message and bloom queries honcho